Source/Core/Core/PowerPC/Jit64/JitAsm.cpp

```diff
@@ -94,9 +94,10 @@ void Jit64AsmRoutineManager::Generate()
     MOV(64, R(RSCRATCH), ImmPtr(CPU::GetStatePtr()));
     TEST(32, MatR(RSCRATCH), Imm32(static_cast<u32>(CPU::State::Stepping)));
     FixupBranch notStepping = J_CC(CC_Z);
-    ABI_PushRegistersAndAdjustStack({}, 0);
-    ABI_CallFunction(PowerPC::CheckBreakPoints);
-    ABI_PopRegistersAndAdjustStack({}, 0);
+    // Causes duplicate hits with jit.cpp
+    // ABI_PushRegistersAndAdjustStack({}, 0);
+    // ABI_CallFunction(PowerPC::CheckBreakPoints);
+    // ABI_PopRegistersAndAdjustStack({}, 0);
```

Comment:
Looks like this was [issue #13044](https://bugs.dolphin-emu.org/issues/13044) that was fixed upstream in [b9aed428](https://github.com/dolphin-emu/dolphin/commit/b9aed428e892b11aacd94b603f05fb412bd12274). It's fine to merge then, and your fix here will automatically disappear the next time I merge upstream

Reply:
Yeah. I checked the official fix and it's what we needed.

Reply:
great, thanks!